src/dao/impl/CatDaoImpl.java

```diff
@@ -0,0 +1,23 @@
+package dao.impl;
+
+import dao.CatDao;
+import repository.entity.CatEntity;
+
+import java.util.UUID;
+
+public class CatDaoImpl implements CatDao {
+    
+    public static CatEntity catEntity;
```

Comment:
Make it private, pls
Otherwise, catEntity might be accessed directly from other places